config/environments/development.rb

```diff
@@ -26,6 +26,7 @@
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = true
+  config.action_controller.asset_host = ENV.fetch('MAS_CMS_PUBLIC_URL', 'http://localhost')
```

Comment:
Ok. We already have on the .env file an env var called MAS_CMS_URL. Is this were you're looking for?